File: 3055-maximum-odd-binary-number/maximum-odd-binary-number.java
class Solution {
    public String maximumOddBinaryNumber(String s) {
       char a[] = s.toCharArray();
       int c1=0;
       int c2=0;
       for(char i : a)
       if(i=='1')
       c1++;
       else
       c2++;
       String res="";
       while(c1!=1)
       {
       res+=1;
       c1--;
       }
       while(c2!=0)
       {
       res+=0;
       c2--;}
       return res+1;
    }
}

File: 3055-maximum-odd-binary-number/maximum-odd-binary-number.py
class Solution:
    def maximumOddBinaryNumber(self, s: str) -> str:
        l=list(s)
        l.sort()
        c=[]
        res=[]
        c.append(l.count('0'))
        c.append(l.count('1'))
        print(l,"ff",c)
        a=int(c[1])
        b=int(c[0])
        while a!=1:
            res.append(1)
            a-=1
        while b!=0:
            res.append(0)
            b-=1
        res.append(1)
        s=""
        for i in res:
            s+=str(i)
        return s